templates/taxi/driver_list.html

```diff
@@ -6,6 +6,9 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
+          {% if user.username == driver.username %}
```

Comment:
what if there were two drivers with the same username?

Reply:
I tested it, username should be unique.